modules/rostests/apitests/CMakeLists.txt

```diff
@@ -1,3 +1,6 @@
+
+PROJECT(ApiTests)
```

Comment:
This is a good idea, but you might need to build them all in the full solution first to get all dependencies built